ryot_compass/src/bevy_compass/drawing/delete.rs

```diff
@@ -19,36 +19,45 @@ pub fn erase_on_click() -> SystemConfigs {
 
 /// A function that listens to the right mouse button and deletes the content of the tile under the cursor.
 /// It always delete the topmost content of the tile, following the Z-ordering.
```

Comment:
this docstring is out of date

Reply:
Good point I need to review the whole documentation, I'll do it